examples/java/src/main/java/org/apache/beam/examples/KafkaWordCountJson.java

```diff
@@ -32,14 +32,14 @@
 //     - strings
 //     - emulator
 //   emulators:
```

Comment:
Using `kafka` as a mapping key is questionable, since it is a `type` of emulator